.github/workflows/lint.yaml

```diff
@@ -37,6 +34,16 @@ jobs:
         # We have to add the current directory as a safe directory or else git commands will not work as expected.
         run: git config --global --add safe.directory $( realpath . ) && git diff --exit-code -- go.mod go.sum api/go.mod api/go.sum
 
+      - name: Set linter versions
+        run: |
+          echo BUF_VERSION=$(cd build.assets; make print-buf-version) >> $GITHUB_ENV
+          echo GOLANGCI_LINT_VERSION=$(cd build.assets; make print-golangci-lint-version) >> $GITHUB_ENV
```

Comment:
```suggestion
          echo BUF_VERSION=$(make -C build.assets print-buf-version) >> $GITHUB_ENV
          echo GOLANGCI_LINT_VERSION=$(make -C build.assets print-golangci-lint-version) >> $GITHUB_ENV
```

Reply:
I had problems with `make` printing "changing to directory $foo" and then "changing to directory $bar" in-between the version number, that's why I switched to the `cd`. 🤷

Reply:
Sounds good.

Reply:
FYI  `make --no-print-directory -C <dir>` will suppress printing "changing to directory...", but I'm fine with a `cd` too,

Reply:
Yep, a `cd` seemed simpler than all of that. Thanks for the tips.